app/scripts/keyrings/hd.js

```diff
@@ -38,10 +38,10 @@ class HdKeyring extends EventEmitter {
     }
 
     if ('numberOfAccounts' in opts) {
```

Comment:
I reccomend using `if (opts.numberOfAccounts) {...}` bc it tells us if that value is truthy, as opposed to just present ( can be undefined and still present )
```js
z={a:undefined}; 'a' in z //=> true
```